tests/vmi_lifecycle_test.go

```diff
@@ -872,7 +872,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 		})
 
-		Context("[Serial]with default cpu model", Serial, func() {
+		Context("[Serial]with default cpu model", Serial, decorators.WgS390x, func() {
```

Comment:
We must have s390x for these tests?

Reply:
Hi @Barakmor1 ,Actually, I am planning to run these tests for s390x. So in the CI we created a label called wgS390x to run the tests which are supported by s390x. So I am adding the WgS390x label here.

Reply:
@fossedihelm, not related to this PR but aren't decorators meant for stating what is required for tests?

Reply:
@Barakmor1 It is not only about what is required but also about categorization, e.g sig-compute,... So here we have a label for tests which are enabled for S390x.

Reply:
I think we should separate those decorators or hav some kind of convention to avoid confusion about what is required and what is categorized.